src/main/java/net/pms/network/HttpException.java

```diff
@@ -1,9 +1,9 @@
 package net.pms.network;
 
-import org.jboss.netty.handler.codec.http.HttpResponseStatus;
+import io.netty.handler.codec.http.HttpResponseStatus;
 
 public class HttpException extends Exception {
-
+	private static final long serialVersionUID = -4952529888336004924L;
```

Comment:
Is this line needed?

Reply:
@SubJunk to use it is recommended by Eclipse -  `The serializable class HttpException does not declare a static final serialVersionUID field of type long`